src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -11,77 +12,69 @@ import * as hooks from './hooks';
 import messages from './messages';
 import * as editorHooks from '../EditorContainer/hooks';
 
-export const VideoUploader = ({ onUpload, errorMessage }) => {
-  const { textInputValue, setTextInputValue } = hooks.uploader();
-  const onURLUpload = hooks.onVideoUpload();
+const URLUploader = () => (
+  <div className="d-flex flex-column flex-wrap">
+    <div className="justify-content-center align-self-center bg-light rounded-circle p-4">
+      <Icon src={FileUpload} className="text-muted" />
+    </div>
+    <div className="d-flex align-self-center justify-content-center flex-wrap flex-column pt-5">
+      <span style={{ fontSize: '20px' }}><FormattedMessage {...messages.dropVideoFileHere} /></span>
+      <span className="align-self-center" style={{ fontSize: '12px' }}><FormattedMessage {...messages.info} /></span>
+    </div>
+    <div className="align-self-center justify-content-center mx-2 text-dark">OR</div>
+  </div>
+);
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({
-    accept: 'video/*',
-    multiple: false,
-    onDrop: (acceptedFiles) => {
-      if (acceptedFiles.length > 0) {
-        const uploadfile = acceptedFiles[0];
-        onUpload(uploadfile);
-      }
-    },
-  });
+export const VideoUploader = ({ onUpload }) => {
+  const { textInputValue, settextInputValue } = hooks.uploader();
+  const onURLUpload = hooks.onVideoUpload();
```

Comment:
This is about onVideoUpload. 

That function is doing things it shouldn't. 

1. Instead of importing store directly you can use the `useStore` hook. 
2. If you want to pass the store as a function argument, just call that function with `useSelector` i.e. `useSelector(postUploadRedirect`. 

I think what you should do instead is, call the selector `postUploadRedirectUrl` as follows:

```
const postUploadRedirectUrl = createSelector([learningContextId, blockId], (contextId, blockId) =>`/course/${learningContextId}/editor/video/${blockId}?selectedVideoUrl=${videoUrl}`). 
```
Then use it as:
```
const redirectUrl = useSelector(postUploadRedirectUrl);
...
// In some component:
//onClick={() => navigateTo(redirectUrl)
```